src/generate-note-controller.js

```diff
@@ -184,6 +184,7 @@ module.exports = function(scribe){
         }
         var selectionIsCollapsed = (markers.length === 1);
 
+        /*
```

Comment:
If this is what @alastairjardine wants then I'll totally remove it. Im leaving this now so it can easily be added in or modified.